### PR TITLE
Adding Royal Canadian Legion

### PR DESCRIPTION
**Pull request by bkissin**
I think I got the syntax correct on this one.

### CODE REVIEW

File: brands/amenity/social_centre.json [hunk 1]
```diff
@@ -63,6 +63,15 @@
       "official_name": "Independent Order of Odd Fellows"
     }
   },
+  "amenity/social_centre|Royal Canadian Legion" : {
+    "countrycodes": ["ca"],
```
Comment (ENT8R) [inline brands/amenity/social_centre.json]:
```suggestion
    "countryCodes": ["ca"],
```

Comment (ENT8R) [reply brands/amenity/social_centre.json]: BTW: by running `npm run build` after you made the changes to the file, you get a direct "feedback" whether the data is correct or not. And it does also format the edited file to be consistent with the other data (e.g. it sorts the entries alphabetically)

Comment (mds08011) [reply brands/amenity/social_centre.json]: @bkissin If you're editing in a browser via Github.com (I am guessing you are), don't worry about the `npm run build` suggestion but do look to see if your Pull Request passes or fails the automated “checks”.  If you see at the bottom of the PR that “Travis CI build failed” this likely indicates that you made a syntax error.

**Comment by bhousel** [comment]
I merged and cleaned up the syntax errors and missing tags in f6b3bef00  
(also added `short_name` tags in this file where appropriate)

**Pull request merged**